pkg/templates/notificationtemplates/notification_generator_test.go

```diff
@@ -0,0 +1,103 @@
+package notificationtemplates_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/pkg/apis/toolchain/v1alpha1"
+	"github.com/codeready-toolchain/host-operator/pkg/apis"
+	"github.com/codeready-toolchain/host-operator/pkg/templates/notificationtemplates"
+	testsupport "github.com/codeready-toolchain/toolchain-common/pkg/test"
+
+	"github.com/pkg/errors"
+	uuid "github.com/satori/go.uuid"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/kubernetes/scheme"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+)
+
+func TestCreateOrUpdateResources(t *testing.T) {
+
+	s := scheme.Scheme
+	err := apis.AddToScheme(s)
+	require.NoError(t, err)
+	logf.SetLogger(zap.Logger(true))
+
+	t.Run("ok", func(t *testing.T) {
+		t.Run("create only", func(t *testing.T) {
```

Comment:
are you also going to address the template updates in this PR (you'll probably need a `test` Asset for that)?

Reply:
yeah, i should have if we were going to keep the create/update code. i guess since we are just returning the NotificationTemplates directly now we can mark this comment as resolved?